.github/workflows/git-artifacts.yml

```diff
@@ -184,7 +199,29 @@ jobs:
           git config --global user.email "<${info#*<}"
         env:
           GPGKEY: ${{secrets.GPGKEY}}
-      - name: Build mingw-w64-${{matrix.arch.name}}-git
+      # Until there is a Git SDK for arm64, we'll need to install a few packages manually
+      # We install prebuilt binaries to save lots of CI time
+      - name: Install aarch64 deps
+        if: env.SKIP != 'true' && matrix.arch.name == 'aarch64'
+        shell: bash
+        run: |
+          set -x
+
+          package_dir="tmp-aarch64-deps"
+          release_url="https://github.com/dennisameling/git/releases/download/v2.39.0.windows.99"
+          packages="mingw-w64-clang-aarch64-openssl-1.1.1.s-1-any.pkg.tar.zst mingw-w64-clang-aarch64-curl-7.86.0-1-any.pkg.tar.zst"
+
+          mkdir -p $package_dir && cd $package_dir
+
+          for package in $packages
+          do
+            curl -LOf $release_url/$package || exit 1
+          done
+
+          pacman -U --noconfirm $packages
+
+          cd ../
```

Comment:
Related discussion for an ARM64 SDK and/or Pacman repo: https://github.com/git-for-windows/git/discussions/4186